modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java

```diff
@@ -298,11 +298,21 @@ public static final class Constants {
         public static final String RECOVERY_ZONE_NAMES_OPTION = "--zones";
 
         public static final String RECOVERY_ZONE_NAMES_OPTION_DESC = "Names specifying zones to get partition states from. "
-                + "Case-sensitive, all zones if not set";
+                + "Case-sensitive, without quotes, all zones if not set";
+
+        public static final String RECOVERY_ZONE_NAME_OPTION = "--zone";
+
+        public static final String RECOVERY_ZONE_NAME_OPTION_DESC = "Name of the zone to reset partitions of. "
+                + "Case-sensitive, without quotes";
```

Comment:
what do you mean by "without quotes"?

Reply:
In SQL commands user has to encapsulate names in quotes if they are not in upper_case ("zone_name"), but in REST api (therefore CLI) we decided not to use quotes